pyproject.toml

```diff
@@ -87,8 +87,7 @@ turbodbc = ["turbodbc"]
 "exa.turbodbc" = "sqlalchemy_exasol.turbodbc:EXADialect_turbodbc"
 
 [tool.pytest.ini_options]
-addopts= "--tb native -v -r fxX"
-python_files= "test/*test_*.py"
+addopts= "--tb native -v -r fxX --log-debug=sqlalchemy.engine --log-debug=sqlalchemy.pool"
```

Comment:
Do we have a description what these parameters mean?

Reply:
This are options documented and provided by the sqla pytest extension. Would you be in favor of removing them from the general settings?

Reply:
I am find with them for this project, I only would add a comment where they are from

Reply:
Created a ticket https://github.com/exasol/sqlalchemy-exasol/issues/219